package.json

```diff
@@ -87,6 +87,7 @@
     "stylelint": "8.4.0",
     "stylelint-config-standard": "18.0.0",
     "textcomplete": "0.16.0",
-    "webpack": "3.10.0"
+    "webpack": "3.10.0",
+    "webpack-dashboard": "1.0.2"
```

Comment:
Their terminal UI sure looks nice, but damn that was long and painful to install:

```js
$ npm i

> farmhash@1.2.1 install .../lounge/node_modules/farmhash
> node-gyp rebuild

  CXX(target) Release/obj.target/farmhash-legacy/src/upstream/farmhash-legacy.o
../src/upstream/farmhash-legacy.cc:684:23: warning: unused function 'Shuffle2031' [-Wunused-function]
STATIC_INLINE __m128i Shuffle2031(__m128i x) {
                      ^
1 warning generated.
  CXX(target) Release/obj.target/farmhash-legacy/src/bindings.o
  SOLINK_MODULE(target) Release/farmhash-legacy.node
clang: warning: libstdc++ is deprecated; move to libc++ with a minimum deployment target of OS X 10.9
  CXX(target) Release/obj.target/farmhash/src/upstream/farmhash.o
  CXX(target) Release/obj.target/farmhash/src/bindings.o
  SOLINK_MODULE(target) Release/farmhash.node
clang: warning: libstdc++ is deprecated; move to libc++ with a minimum deployment target of OS X 10.9

> integer@1.0.1 install .../lounge/node_modules/integer
> node tools/install

  CXX(target) Release/obj.target/integer/src/integer.o
  SOLINK_MODULE(target) Release/integer.node
clang: warning: libstdc++ is deprecated; move to libc++ with a minimum deployment target of OS X 10.9

> better-sqlite3@4.0.3 install .../lounge/node_modules/better-sqlite3
> node deps/install

==> cwd: .../lounge/node_modules/better-sqlite3
==> .../lounge/node_modules/lzz-gyp/lzz-compiled/osx -hx hpp -sx cpp -k BETTER_SQLITE3 -d -hl -sl -e ./src/better_sqlite3.lzz
==> cwd: .../lounge/node_modules/better-sqlite3
==> node-gyp rebuild
  ACTION deps_sqlite3_gyp_action_before_build_target_unpack_sqlite_dep Release/obj/gen/sqlite-autoconf-3190300/sqlite3.c
  TOUCH Release/obj.target/deps/action_before_build.stamp
  CC(target) Release/obj.target/sqlite3/gen/sqlite-autoconf-3190300/sqlite3.o
  LIBTOOL-STATIC Release/sqlite3.a
  CXX(target) Release/obj.target/better_sqlite3/src/better_sqlite3.o
  SOLINK_MODULE(target) Release/better_sqlite3.node
clang: warning: libstdc++ is deprecated; move to libc++ with a minimum deployment target of OS X 10.9
  CC(target) Release/obj.target/test_extension/deps/test_extension.o
  SOLINK_MODULE(target) Release/test_extension.node
clang: warning: libstdc++ is deprecated; move to libc++ with a minimum deployment target of OS X 10.9
thelounge@2.7.0-pre.4 .../lounge
└─┬ webpack-dashboard@1.0.2
  ├── blessed@0.1.81
  ├── cross-spawn@4.0.2
  ├── filesize@3.5.11
  ├─┬ inspectpack@2.2.2
  │ ├─┬ better-sqlite3@4.0.3
  │ │ ├── bindings@1.3.0
  │ │ ├── integer@1.0.1
  │ │ ├── lzz-gyp@0.4.2
  │ │ └── to-descriptor@1.0.1
  │ ├── farmhash@1.2.1
  │ ├─┬ uglify-es@3.3.4
  │ │ └── source-map@0.6.1
  │ ├── workerpool@2.3.0
  │ └─┬ yargs@4.8.1
  │   ├── cliui@3.2.0
  │   ├── os-locale@1.4.0
  │   ├─┬ read-pkg-up@1.0.1
  │   │ ├─┬ find-up@1.1.2
  │   │ │ └── path-exists@2.1.0
  │   │ └─┬ read-pkg@1.1.0
  │   │   ├─┬ load-json-file@1.1.0
  │   │   │ ├── parse-json@2.2.0
  │   │   │ ├── pify@2.3.0
  │   │   │ └── strip-bom@2.0.0
  │   │   └── path-type@1.1.0
  │   ├─┬ string-width@1.0.2
  │   │ └── is-fullwidth-code-point@1.0.0
  │   ├── which-module@1.0.0
  │   ├── window-size@0.2.0
  │   └─┬ yargs-parser@2.4.1
  │     └── camelcase@3.0.0
  ├─┬ most@1.7.2
  │ ├── @most/multicast@1.3.0
  │ ├── @most/prelude@1.7.0
  │ └── symbol-observable@1.1.0
  ├─┬ socket.io@1.7.4
  │ ├─┬ debug@2.3.3
  │ │ └── ms@0.7.2
  │ ├─┬ engine.io@1.8.5
  │ │ ├── accepts@1.3.3
  │ │ ├─┬ engine.io-parser@1.3.2
  │ │ │ └── wtf-8@1.0.0
  │ │ └─┬ ws@1.1.5
  │ │   ├── options@0.0.6
  │ │   └── ultron@1.0.2
  │ ├─┬ has-binary@0.1.7
  │ │ └── isarray@0.0.1
  │ ├── object-assign@4.1.0
  │ ├── socket.io-adapter@0.5.0
  │ └─┬ socket.io-parser@2.3.1
  │   ├── component-emitter@1.1.2
  │   ├─┬ debug@2.2.0
  │   │ └── ms@0.7.1
  │   └── isarray@0.0.1
  └─┬ socket.io-client@1.7.4
    ├── component-emitter@1.2.1
    └─┬ engine.io-client@1.8.5
      ├── component-emitter@1.2.1
      ├── parsejson@0.0.3
      └── xmlhttprequest-ssl@1.5.3
```

Reply:
Running this, I see a couple issues:

- Since they show the log tab as a kinda-emulated terminal, they have to recreate a scrollbar mechanism and... it's not going too well (at least on macOS). It's not smooth at all, so scrolling a tiny bit goes all the way up. On my laptop (13" with terminal in full screen), the "Log" tab is only 12 lines high and most lines have an extra new line between them, so I constantly have to try and scroll. Not a huge thing, but an annoying inconvenience if this is something meant to help developing/debugging.

- In "Logs" tab, Webpack outputs a bunch of "Worker error Error: No code sections found". This doesn't appear with `npm run build` or `npm run watch`, odd.

- ... I wanted to copy-paste the whole output, but (at least on macOS / iTerm2, haven't tried other OS) text cannot be selected, so cannot be copied to clipboard. I think that's an issue, but maybe there are ways to "tweak the UI"? Haven't looked.

Reply:
Actually, I can repro on `npm run build` if I remove the `if` around the plugin definition in Webpack config (i.e. always add the plugin):

```js
$ npm run build

> thelounge@2.7.0-pre.4 build .../lounge
> webpack

   332 modules
Worker error Error: No code sections found
    at Bundle.validate (.../lounge/node_modules/inspectpack/lib/models/bundle.js:54:11)
    at createBundle (.../lounge/node_modules/inspectpack/lib/models/bundle.js:262:12)
    at Immediate.setImmediate (.../lounge/node_modules/inspectpack/lib/models/bundle.js:313:7)
    at runCallback (timers.js:672:20)
    at tryOnImmediate (timers.js:645:5)
    at processImmediate [as _immediateCallback] (timers.js:617:5)
Worker error Error: No code sections found
    at Bundle.validate (.../lounge/node_modules/inspectpack/lib/models/bundle.js:54:11)
    at createBundle (.../lounge/node_modules/inspectpack/lib/models/bundle.js:262:12)
    at Immediate.setImmediate (.../lounge/node_modules/inspectpack/lib/models/bundle.js:313:7)
    at runCallback (timers.js:672:20)
    at tryOnImmediate (timers.js:645:5)
    at processImmediate [as _immediateCallback] (timers.js:617:5)
Worker error Error: No code sections found
    at Bundle.validate (.../lounge/node_modules/inspectpack/lib/models/bundle.js:54:11)
    at createBundle (.../lounge/node_modules/inspectpack/lib/models/bundle.js:262:12)
    at Immediate.setImmediate (.../lounge/node_modules/inspectpack/lib/models/bundle.js:313:7)
    at runCallback (timers.js:672:20)
    at tryOnImmediate (timers.js:645:5)
    at processImmediate [as _immediateCallback] (timers.js:617:5)
Worker error Error: No code sections found
    at Bundle.validate (.../lounge/node_modules/inspectpack/lib/models/bundle.js:54:11)
    at createBundle (.../lounge/node_modules/inspectpack/lib/models/bundle.js:262:12)
    at Immediate.setImmediate (.../lounge/node_modules/inspectpack/lib/models/bundle.js:313:7)
    at runCallback (timers.js:672:20)
    at tryOnImmediate (timers.js:645:5)
    at processImmediate [as _immediateCallback] (timers.js:617:5)
Worker error Error: No code sections found
    at Bundle.validate (.../lounge/node_modules/inspectpack/lib/models/bundle.js:54:11)
    at createBundle (.../lounge/node_modules/inspectpack/lib/models/bundle.js:262:12)
    at Immediate.setImmediate (.../lounge/node_modules/inspectpack/lib/models/bundle.js:313:7)
    at runCallback (timers.js:672:20)
    at tryOnImmediate (timers.js:645:5)
    at processImmediate [as _immediateCallback] (timers.js:617:5)
Worker error Error: No code sections found
    at Bundle.validate (.../lounge/node_modules/inspectpack/lib/models/bundle.js:54:11)
    at createBundle (.../lounge/node_modules/inspectpack/lib/models/bundle.js:262:12)
    at Immediate.setImmediate (.../lounge/node_modules/inspectpack/lib/models/bundle.js:313:7)
    at runCallback (timers.js:672:20)
    at tryOnImmediate (timers.js:645:5)
    at processImmediate [as _immediateCallback] (timers.js:617:5)
```